tests/e2e/helpers.go

```diff
@@ -1636,3 +1640,64 @@ func getReadyProfileBundle(t *testing.T, f *framework.Framework, name, namespace
 
 	return pb, nil
 }
+
+func writeToArtifactsDir(t *testing.T, f *framework.Framework, scan, pod, container string, log string) error {
+	dir := os.Getenv("ARTIFACT_DIR")
```

Comment:
This is a nit, but should we continue if the environmental var is not set and log to cwd or set up some other directory to put the logs to? Having the logs in cwd is slightly irritating for a local scan :-)